ready-api-testserver-client-core/src/main/java/com/smartbear/readyapi/client/execution/TestServerRecipeExecutionResult.java

```diff
@@ -0,0 +1,18 @@
+package com.smartbear.readyapi.client.execution;
+
+import com.smartbear.readyapi.client.model.ProjectResultReport;
+import com.smartbear.readyapi.client.result.AbstractRecipeExecutionResult;
+import com.smartbear.readyapi.client.result.TestServerTestStepResult;
+
+public class TestServerRecipeExecutionResult extends AbstractRecipeExecutionResult {
```

Comment:
We should really have at least class-level Javadoc everywhere. Remember this is supposed to be a public API; that makes it even more important!